modules/ROOT/partials/integrations/jquery-quick-start.adoc

```diff
@@ -83,7 +83,7 @@ API key, which is created when signing up to the link:{accountsignup}/[{cloudnam
 +
 [source,html]
 ----
-<script src="https://cdn.jsdelivr.net/npm/@tinymce/tinymce-jquery@1/dist/tinymce-jquery.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/@tinymce/tinymce-jquery@2/dist/tinymce-jquery.min.js"></script>
```

Comment:
I wonder if it's worth making this an attribute as well? Up to you though as it's not used often.

Reply:
done